geomesa-fs/geomesa-fs-storage/geomesa-fs-storage-parquet/src/main/scala/org/locationtech/geomesa/parquet/io/SimpleFeatureParquetSchema.scala

```diff
@@ -207,8 +207,8 @@ object SimpleFeatureParquetSchema {
             .requiredList().requiredListElement().element(PrimitiveTypeName.DOUBLE, Repetition.REPEATED).named(GeometryColumnX)
             .requiredList().requiredListElement().element(PrimitiveTypeName.DOUBLE, Repetition.REPEATED).named(GeometryColumnY)
 
-      case ObjectType.GEOMETRY_COLLECTION =>
```

Comment:
Since we are dropping this clear warning/error, we may want to toss in a GeometryCollection in a unit test and see what happens:).

Reply:
yeah i think geometry collection support is not consistent atm...

Reply:
Given that, should we do anything particular about it?  Docs?  Add in cases like this?

Ignore that case since people would be crazy to have a column of GeometryCollections?!  (I'm fine with anything.)